src/explore-education-statistics-admin/src/pages/release/content/components/ReleasePreviewTableTool.tsx

```diff
@@ -17,16 +17,18 @@ import React, { useState } from 'react';
 import { ReleaseType } from '@common/services/types/releaseType';
 
 interface Props {
+  featuredTableId?: string;
   releaseId: string;
   releaseType: ReleaseType;
   publication: Publication | ContentPublication;
 }
 const ReleasePreviewTableTool = ({
+  featuredTableId,
   releaseId,
   releaseType,
   publication,
 }: Props) => {
-  const [dataBlockId, setDataBlockId] = useState<string>();
+  const [dataBlockId, setDataBlockId] = useState<string>(featuredTableId ?? '');
```

Comment:
Don't need the `<string>` if you're declaring the default value here, I don't think.

Reply:
Good spot, I've removed it.